homeassistant/components/solax/sensor.py

```diff
@@ -108,7 +108,8 @@ async def async_setup_entry(
     for sensor, (idx, measurement) in api.inverter.sensor_map().items():
         description = SENSOR_DESCRIPTIONS[(measurement.unit, measurement.is_monotonic)]
 
-        uid = f"{serial}-{idx}"
+        postfix = sensor.replace(" ", "").lower()
+        uid = f"{serial}-{idx}-{postfix}"
```

Comment:
Changing the unique_id is a breaking change. That should be avoided or migrated for existing entities.

If the sensor that creates duplicate unique_id is redundant, maybe it's easier to skip that measurement?